src/components/containers/headers/userHeader.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import UserHeaderView from "../../views/headers/UserHeader";
+
+class UserHeader extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+    };
+  }
```

Comment:
I don't think we need `this.state `object, since it's not used in this class based function